frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -59,10 +59,11 @@ function isBreakdownFunnelResults(results: FunnelStep[] | FunnelStep[][]): resul
     return Array.isArray(results) && (results.length === 0 || Array.isArray(results[0]))
 }
 
+// breakdown parameter could be a string (property breakdown) or object/number (list of cohort ids)
 function isValidBreakdownParameter(
     breakdown: FunnelRequestParams['breakdown']
 ): breakdown is string | null | undefined {
```

Comment:
This type guard needs to be updated.

Reply:
not sure if this is right but i removed it because it doesn't change the type now